src/libs/ReportActionsUtils.ts

```diff
@@ -320,7 +320,7 @@ function getCombinedReportActions(reportActions: ReportAction[], transactionThre
     // Filter out request and send money request actions because we don't want to show any preview actions for one transaction reports
     const filteredReportActions = [...reportActions, ...filteredTransactionThreadReportActions].filter((action) => {
         const actionType = (action as OriginalMessageIOU).originalMessage?.type ?? '';
-        return actionType !== CONST.IOU.REPORT_ACTION_TYPE.CREATE && actionType !== CONST.IOU.REPORT_ACTION_TYPE.TRACK && !isSentMoneyReportAction(action);
+        return actionType !== CONST.IOU.REPORT_ACTION_TYPE.CREATE && !isSentMoneyReportAction(action);
```

Comment:
We should not remove it directly.

Removing it breaks shows the report action in transaction report.

https://github.com/Expensify/App/assets/102477862/f252a9ea-2fcf-4e65-814f-1043d6a57725

We should not filter track actions only in self DMs as far as I see. The following snippet fixes our bug and appears to cause no regressions.
```suggestion
       const report = allReports?.[action?.reportID ?? ''];
        const isSelfDM = report?.chatType === CONST.REPORT.CHAT_TYPE.SELF_DM;
        if (isSelfDM) {
             return actionType !== CONST.IOU.REPORT_ACTION_TYPE.CREATE && !isSentMoneyReportAction(action);
        }
        return actionType !== CONST.IOU.REPORT_ACTION_TYPE.CREATE && actionType !== CONST.IOU.REPORT_ACTION_TYPE.TRACK && !isSentMoneyReportAction(action);
```